src/components/Login/ForgotPasswordSubmit/index.js

```diff
@@ -40,7 +40,7 @@ const ForgotPasswordSubmit = ({ history, error, username, forgotPasswordSubmit }
           </button>
         </div>
         <p>fghjk{error}</p>
-        <ErrorMsgBox error={error} />
+        <MessageBox error={error} />
```

Comment:
Looks like you missed `type` here?

Reply:
type has a default value in Message.defaultProps